addons/mail/static/src/messaging/component/composer_text_input/composer_text_input.js

```diff
@@ -161,14 +282,82 @@ class ComposerTextInput extends Component {
      */
     _onKeydownTextarea(ev) {
         switch (ev.key) {
+            // UP, DOWN, TAB: prevent moving cursor if navigation in mention propositions
+            case 'ArrowUp':
+            case 'ArrowDown':
+            case 'Tab':
+                if (this.state.isMentionsOpen) {
+                    ev.preventDefault();
+                }
+                break;
+            // ENTER: submit the message only if the dropdown mention proposition is not displayed
             case 'Enter':
-                this._onKeydownTextareaEnter(ev);
+                if (this.state.isMentionsOpen) {
+                    ev.preventDefault();
+                } else {
+                    this._onKeydownTextareaEnter(ev);
```

Comment:
same comment as with `_onKeyupTextareaEscape` (include logic of mention open in the function)